packages/server/src/api/muster/muster.controller.ts

```diff
@@ -130,56 +127,56 @@ class MusterController {
     res.json(deletedMusterConfig);
   }
 
-  async getMusterRoster(req: ApiRequest<OrgRoleParams, null, GetMusterRosterQuery>, res: Response<Paginated<Partial<Roster>>>) {
-    assertRequestQuery(req, [
-      'fromDate',
-      'toDate',
-      'limit',
-      'page',
-    ]);
-
-    const fromDate = moment(req.query.fromDate);
-    const toDate = moment(req.query.toDate);
-    const limit = parseInt(req.query.limit);
-    const page = parseInt(req.query.page);
-    const unitId = (req.query.unitId != null) ? parseInt(req.query.unitId) : undefined;
-
-    const rosterStats = await getMusterRosterStats({
-      org: req.appOrg!,
-      userRole: req.appUserRole!,
-      unitId,
-      fromDate,
-      toDate,
-    });
-
-    const offset = page * limit;
-
-    return res.json({
-      rows: rosterStats.slice(offset, offset + limit),
-      totalRowsCount: rosterStats.length,
-    });
-  }
-
-  async getMusterUnitTrends(req: ApiRequest<null, null, GetMusterUnitTrendsQuery>, res: Response) {
-    assertRequestQuery(req, [
-      'currentDate',
-      'weeksCount',
-      'monthsCount',
-    ]);
-
-    const currentDate = moment(req.query.currentDate);
-    const weeksCount = parseInt(req.query.weeksCount ?? '6');
-    const monthsCount = parseInt(req.query.monthsCount ?? '6');
-
-    const unitTrends = await getMusterUnitTrends({
-      userRole: req.appUserRole!,
-      currentDate,
-      weeksCount,
-      monthsCount,
-    });
-
-    res.json(unitTrends);
-  }
+  // async getMusterRoster(req: ApiRequest<OrgRoleParams, null, GetMusterRosterQuery>, res: Response<Paginated<Partial<Roster>>>) {
```

Comment:
Hmmm....Not sure why this is commented out? What is the implication with workspaces here?

Reply:
I think a lot of stuff in this PR won't make sense until the muster compliance task is finished.  I have re-written most of the muster compliance logic.

Reply:
alright I'll defer to @jdgarrett then since he is working that too

Reply:
alright I'll defer to @jdgarrett then since he is working that too